src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -482,7 +482,7 @@ private void randoopConsistencyTest2(Sequence newSequence) {
 
   // Checks that the set allSequencesAsCode contains a set of strings
   // equivalent to the sequences in allSequences.
-  private void randoopConsistencyTests(Sequence newSequence) {
+  protected void randoopConsistencyTests(Sequence newSequence) {
```

Comment:
Why has this been changed from private to protected?  It doesn't seem to be called anywhere except from this file.
Can you either revert all these changes (which will make the diffs smaller), or create a separate pull request so that each pull request has a single purpose and can have a descriptive commit message?